pyvista/plotting/plotting.py

```diff
@@ -1508,6 +1524,11 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
 
         rgb = kwargs.pop('rgba', rgb)
 
+        # account for legacy behavior
+        if 'stitle' in kwargs:  # pragma: no cover
+            warnings.warn(DeprecationWarning(USE_SCALAR_BAR_ARGS))
```

Comment:
Bikeshedding: although functionally the same, all the examples around https://docs.python.org/3/library/warnings.html#warnings.warn suggest to use `warnings.warn(USE_SCALAR_BAR_ARGS, DeprecationWarning)`. But the current version is also documented, so it's OK to keep it. (If changed, the other one should be changed too.)

Reply:
https://github.com/pyvista/pyvista/pull/1233/commits/48464cca32ca677e75a052b1dfa2e24983c26dc1